packages/beacon-node/src/network/reqresp/handlers/lightClientUpdate.ts

```diff
@@ -0,0 +1,17 @@
+import {altair} from "@lodestar/types";
+import {MAX_REQUEST_LIGHT_CLIENT_UPDATES} from "@lodestar/params";
+import {IBeaconChain} from "../../../chain/index.js";
+import {ResponseError} from "../response/index.js";
+import {RespStatus} from "../../../constants/index.js";
+
+export async function* onLightclientUpdate(
+  requestBody: altair.LightClientUpdatesByRange,
+  chain: IBeaconChain
+): AsyncIterable<altair.LightClientUpdate> {
+  try {
+    const count = Math.min(MAX_REQUEST_LIGHT_CLIENT_UPDATES, requestBody.count);
+    yield* await chain.lightClientServer.getUpdates(requestBody.startPeriod, count);
+  } catch (e) {
+    throw new ResponseError(RespStatus.RESOURCE_UNAVAILABLE, (e as Error).message);
```

Comment:
Note: The `chain.lightClientServer.getUpdates` could fail to produce a result for various reasons...this, according to the spec, should lead to an error code 3: ResourceUnavailable. The `(e as Error).message` provides additional textual information what error it was